### ISSUE TITLE
suspending not working

### ISSUE THREAD

**Issue by jinigithub** [jmontleon/pixelbook-fedora#27]
thank you for saving my Pixelbook,all are perfect except in my pixelbook,clicking suspend/closing the lid/click the side button make shutdown at once.
i'm a noob in linux,my pixelbook keyboard is not working in win10/11 althoug win recognize it as pixelbook keyboard and drive it right.
fedora and your work is the last hope,tks

**Comment by jmontleon** [comment]
I'm not sure about the suspend issues.

As for the power button always shutting down immediately see here. My guess is you're using Gnome. Have a look at this issue and the associated PR. I don't think it was documented in the README, although it was addded to the playbook (PR welcome for the docs, otherwise I'll get around to it at some point).

https://github.com/jmontleon/pixelbook-fedora/issues/19

Basically the answer is to:
```
sudo mv /etc/acpi/events/powerconf /etc/acpi/events/powerconf.disabled~
sudo systemctl restart acpid
```

It's possible this will fix your suspend issues as well. Otherwise, ensure you're running the latest version of coreboot (several issues have been fixed by newer versions) and see if that helps.

**Comment by jmontleon** [comment]
It's been over a week since the last response so I'm closing this. If you are still having a problem please respond with what you've tried and we can look for other other things to try.
